handlers/http.go

```diff
@@ -35,8 +39,9 @@ func stringifyRequest(req common.Request) *bytes.Buffer {
 }
 
 type httpRateLimiter struct {
-	rateLimiter ratelimiter.RateLimiter
-	proxy       http.Handler
+	rateLimiter  ratelimiter.RateLimiter
+	proxy        http.Handler
+	AllowOnError bool // Do not limit on errors when true
```

Comment:
No, I needed a way to keep the function interfaces the same, and still set the AllowOnError flag. 

I could have done breaking changes to the function signatures, but this is a public repo, so I figured I'd keep it backwards compat.